File: backend/data-streaming-server/src/main/java/com/insightweb/datastreamingserver/DataStreamingServerApplication.java
package com.insightweb.datastreamingserver;

import com.insightweb.datastreamingserver.usecase.TrackerEventProcessing;
import lombok.RequiredArgsConstructor;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@RequiredArgsConstructor
public class DataStreamingServerApplication implements CommandLineRunner {
	private final TrackerEventProcessing trackerEventProcessing;

	public static void main(String[] args) {
		SpringApplication.run(DataStreamingServerApplication.class, args);
	}

	@Override
	public void run(String... args) throws Exception {
		trackerEventProcessing.startStream();
	}
}

File: backend/data-streaming-server/src/main/kotlin/com/insightweb/datastreamingserver/DataStreamingServerApplication.kt
package com.insightweb.datastreamingserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class DataStreamingServerApplication {
}

fun main(args: Array<String>) {
    runApplication<DataStreamingServerApplication>(*args)
}